pkg/driverbuilder/builder/builders.go

```diff
@@ -389,13 +397,11 @@ func (c Config) toTemplateData(b Builder, kr kernelrelease.KernelRelease) common
 func resolveURLReference(u string) string {
 	uu, err := url.Parse(u)
 	if err != nil {
-		slog.Error(err.Error())
-		os.Exit(1)
+		panic(err)
 	}
 	base, err := url.Parse(uu.Host)
 	if err != nil {
-		slog.Error(err.Error())
-		os.Exit(1)
+		panic(err)
```

Comment:
Panic instead of erroring out. Again, this was done to avoid passing the printer down here.